chalicelib/app.py

```diff
@@ -4,14 +4,31 @@
 from chalicelib.app_utils import AppUtils
 from chalicelib.deploy import Deploy
 
-
 # Chalice metadata
 app = Chalice(app_name='foursight-cgap')
 app.debug = True
 STAGE = os.environ.get('chalice_stage', 'dev')
-DEFAULT_ENV = 'cgap'
+DEFAULT_ENV = os.environ.get("ENV_NAME", "cgap-uninitialized")
 app_utils_obj = AppUtils()
 
+# When running 'chalice local' we do not get the same "/api" prefix as we see when deployed to AWS (Lambda).
+# So we set it explicitly here if your CHALICE_LOCAL environment variable is set.
+# Seems to be a known issue: https://github.com/aws/chalice/issues/838
```

Comment:
Do we also not log in? Is this going to affect the API callbacks used by auth0 or do they not get used? (I don't know the answer to this question, but it was just something it occurred to me to wonder about.

Reply:
Also, will this not also mean that the chalice local is not as useful to test what this will do in the field because it's not doing it the same way? That is, won't it not catch certain classes of problems and force us to catch them non-locally? It would be good to understand that trade-off better, and perhaps to document it here.

Reply:
Running locally there is no login, so Auth0 doesn't get used. You make a good point about being different for local and AWS/in-the-field, at least in terms of not going through the exact code path; but the reason I did this was exactly because I was running into issues where the difference between having the /api in AWS and not locally was impeding my troubleshooting/understanding, so for me it was more important to make local and AWS more alike in this regard.